api/src/pcapi/scripts/index_artists_offers/main.py

```diff
@@ -0,0 +1,59 @@
+"""
+Job console documentation here: https://www.notion.so/passcultureapp/Documentation-Job-Console-769beeacd5a146de9c97b6f8ee544276
+Assumed path to the script (copy-paste in github actions): 
+
+https://github.com/pass-culture/pass-culture-main/blob/pc-35323-reindex-offers-for-artists/api/src/pcapi/scripts/index_artists_offers/main.py
+
+"""
+
+import argparse
+import logging
+
+from sqlalchemy.sql.expression import func
+
+from pcapi.app import app
+from pcapi.core.artist.models import ArtistProductLink
+from pcapi.core.offers.models import Offer
+from pcapi.core.offers.models import Product
+from pcapi.core.search import IndexationReason
+from pcapi.core.search import async_index_offer_ids
+from pcapi.models import db
+
+logger = logging.getLogger(__name__)
+
+
+def main(batch_size: int = 10_000, not_dry: bool = False) -> None:
+    max_id = db.session.query(func.max(ArtistProductLink.id)).scalar()
+    total = 0
+    min_id = 0
+    while min_id <= max_id:
+        query = (
+            db.session.query(ArtistProductLink)
+            .join(Product, ArtistProductLink.product_id == Product.id)
+            .join(Product.offers)
+            .filter(ArtistProductLink.id >= min_id, ArtistProductLink < min_id + batch_size)
+            .with_entities(Offer.id)
+        )
+        for batch in query.yield_per(batch_size):
+            ids = [offer_id for offer_id in batch]
+            logger.info(ids)
```

Comment:
```suggestion
        for (offer_id,) in query.yield_per(batch_size):
            logger.info(offer_id)
```

ta syntaxe laisse penser qu'à chaque élément de `query` on a une liste d'id, mais en fait on a juste un tuple composé d'un seul id